docs/content/basic-concepts.md

```diff
@@ -21,6 +21,18 @@ $ npm install --save spectacle
 
 <a name="development"></a>
 
+## Getting Started with Development
+
+The `src` directory contains all the source for the Spectacle library. All components designed to be part of the Spectacle API must be exported in `src/index.js`.
+
+#### JavaScript-based Decks
+
+To start the development server at port `3000` against a JavaScript-based deck use `yarn start:js` or `npm run start:js`.
+
+#### Markdown-based Decks
+
+To start the development server at port `3100` against a Markdown-based deck use `yarn start:md` or `npm run start:md`. The source for the development decks are in `examples/js` and `examples/md`.
```

Comment:
Maybe place the sentence:

> The source for the development decks are in `examples/js` and `examples/md`.

in each subsection specifically like:

> To start the development server at port `3000` against a JavaScript-based deck (found in `examples/js`) use `yarn start:js` or `npm run start:js`.

and

> To start the development server at port `3100` against a Markdown-based deck (found in `examples/md`) use `yarn start:md` or `npm run start:md`.

?